cmake/packaging.cmake

```diff
@@ -0,0 +1,253 @@
+# We need to define everything for CPack at the top level of the project.
+# Variables defined in CMake files included via add_subdirectory don't make it back to the calling CMake file.
```

Comment:
Are these CPack settings picked up by external projects too? Meaning, projects like Logical Quotas get to delete common CPack instructions shared by all iRODS projects?

Reply:
Nothing in this file is passed down. I am planning on moving more common CPack stuff to a CMake module that downstream projects can pull in, and this will likely make it into my next reorganization PR.